cockatrice/src/interface/widgets/server/game_selector.cpp

```diff
@@ -270,12 +293,20 @@ void GameSelector::customContextMenu(const QPoint &point)
 
     QMenu menu;
     menu.addAction(&joinGame);
+
+    if (tabSupervisor->getUserInfo()->user_level() & ServerInfo_User::IsJudge) {
+        QAction joinGameAsJudge(tr("Join Game as Judge"));
+        connect(&joinGameAsJudge, &QAction::triggered, this, &GameSelector::actJoinAsJudge);
+
+        menu.addAction(&joinGameAsJudge);
+    }
+
     menu.addAction(&spectateGame);
     menu.addAction(&getGameInfo);
     menu.exec(gameListView->mapToGlobal(point));
 }
 
-void GameSelector::joinGame(const bool isSpectator)
+void GameSelector::joinGame(const bool isSpectator, const bool isJudge)
```

Comment:
I think this should be "asJudge", it's a bit confusing when judges join a game as player otherwise